.github/workflows/master_release.yml

```diff
@@ -0,0 +1,72 @@
+name: Master/Release branch workflow
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+
+env:
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_EMAIL: ${{ secrets.BOT_GITHUB_EMAIL }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+
+jobs:
+  run_ci:
+    uses: ./.github/workflows/ci.yml
+    secrets: inherit
+  push_to_nexus:
+    name: "Push to Nexus"
+    runs-on: ubuntu-latest
+    needs: [run_ci]
+    if: >
+      always() &&
+      (needs.run_ci.result == 'skipped' || needs.run_ci.result == 'success') &&
+      (! contains(github.event.head_commit.message, '[no release]')) &&
+      (github.ref_name == 'master' || contains(github.ref_name, 'release/')) &&
```

Comment:
this is redundant (always evaluate to true) if this workflow can be triggered only via master and release/** as per https://github.com/Alfresco/alfresco-community-repo/pull/1631/files#diff-5a9780f69029ac38d1ee9273926bf5f3f20ef130115d8669d413865caf68d0c0R5

Reply:
Removed redundant condition.